rasa/core/train.py

```diff
@@ -108,17 +108,15 @@ async def train_comparison_models(
                 )
 
                 with TempDirectoryPath(tempfile.mkdtemp()) as train_path:
-                    _, new_fingerprint = await asyncio.gather(
-                        train(
-                            domain,
-                            file_importer,
-                            train_path,
-                            policy_config=policy_config,
-                            exclusion_percentage=percentage,
-                            additional_arguments=additional_arguments,
-                        ),
-                        model.model_fingerprint(file_importer),
+                    train(
```

Comment:
this in turn makes `train_comparison_models` and `do_compare_training` sync, right? 🙈

Reply:
Right. Was hard to find all those little things that became sync :( 
But updated it so it should be good now!

Reply:
yeah I figured 😄 hopefully we don't decide to re-add `async` soon 😄